assets/js/components/SaptuneDetails/SaptuneServiceStatus.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { get } from 'lodash';
+import HealthIcon from '@components/Health/HealthIcon';
+
+const servicesIcons = {
+  saptune: {
+    'enabled/active': <HealthIcon health="passing" />,
+    'enabled/inactive': <HealthIcon health="warning" />,
+    'disabled/active': <HealthIcon health="warning" />,
+    'disabled/inactive': <HealthIcon health="critical" />,
+  },
+  sapconf: {
+    'enabled/active': <HealthIcon health="critical" />,
+    'enabled/inactive': <HealthIcon health="warning" />,
+    'disabled/active': <HealthIcon health="critical" />,
+    'disabled/inactive': <HealthIcon health="passing" />,
+  },
+  tuned: {
+    'enabled/active': <HealthIcon health="warning" />,
+    'enabled/inactive': <HealthIcon health="warning" />,
+    'disabled/active': <HealthIcon health="warning" />,
+    'disabled/inactive': <HealthIcon health="passing" />,
+  },
+};
+
+function SaptuneServiceStatus({ service, enabled, active }) {
+  if (!enabled) {
+    return <span>-</span>;
+  }
+
+  const status = `${enabled}/${active}`;
+  const icon = get(servicesIcons, [service.name, status], null);
```

Comment:
`service.name` ?, why don't we pass the `serviceName` directly here?

Reply:
yeah much better!